src/client/scripts/game/chess/pieces.js

```diff
@@ -195,7 +214,25 @@ const pieces = (function () {
             obstaclesN: getSpriteCoords(pieceWidth, 4,7),
 
             // Miscellaneous
-            yellow: getSpriteCoords(pieceWidth, 5,7) // COIN
+            yellow: getSpriteCoords(pieceWidth, 5,7), // COIN
+
+            // 4 player
+            kingsG: getSpriteCoords(pieceWidth, 3,2),
+            kingsU: getSpriteCoords(pieceWidth, 3,2),
+            pawnsG: getSpriteCoords(pieceWidth, 1,1),
+            pawnsU: getSpriteCoords(pieceWidth, 1,1),
+            knightsG: getSpriteCoords(pieceWidth, 3,1),
+            knightsU: getSpriteCoords(pieceWidth, 3,1),
+            bishopsG: getSpriteCoords(pieceWidth, 5,1),
+            bishopsU: getSpriteCoords(pieceWidth, 5,1),
+            rooksG: getSpriteCoords(pieceWidth, 7,1),
+            rooksU: getSpriteCoords(pieceWidth, 7,1),
+            queensG: getSpriteCoords(pieceWidth, 1,2),
+            queensU: getSpriteCoords(pieceWidth, 1,2),
+            royalQueensG: getSpriteCoords(pieceWidth, 3,6),
+            royalQueensU: getSpriteCoords(pieceWidth, 3,6),
+            royalCentaursG: getSpriteCoords(pieceWidth, 1,6),
+            royalCentaursU: getSpriteCoords(pieceWidth, 1,6),
```

Comment:
Adding texture coords to the spritesheet for each color isn't very expandable, I recommend to have the colored pieces fetch the texture coordinate for the white pieces.